scalability_and_performance/ztp-far-edge/ztp-reference-cluster-configuration-for-vdu.adoc

```diff
@@ -0,0 +1,182 @@
+:_content-type: ASSEMBLY
+[id="sno-configure-for-vdu"]
+= Recommended {sno} cluster configuration for vDU application workloads
+include::_attributes/common-attributes.adoc[]
+:context: sno-configure-for-vdu
+
+toc::[]
+
+You must configure the {sno} cluster before you can deploy virtual distributed unit (vDU) applications in the cluster. Changes include cluster optimizations for high performance workloads, enabling workload partitioning, and minimizing the number of reboots required post-installation.
+
+[id="sno-install-time-cluster-config_{context}"]
+== Installation-time cluster configuration
+
+You need to apply cluster configurations during installation so that the cluster meets the feature and performance requirements necessary for running a vDU application.
+
+[id="sno-du-enabling-workload-partitioning_{context}"]
+=== Workload partitioning
+
+During {sno} cluster installation, you must enable workload partitioning. This limits the cores allowed to run platform services, maximizing the CPU core for application payloads.
+
+[NOTE]
+====
+Workload partitioning can only be enabled during cluster installation. You cannot disable workload partitioning post-installation. However, you can reconfigure workload partitioning by updating the `cpu` value that you define in the performance profile, and in the related `MachineConfig` CR.
+====
+
+* The base64-encoded custom resource (CR) that enables workload partitioning contains the CPU set that the management workloads are constrained to. Encode host-specific values for `crio.conf` and `kubelet.conf` in base64. This content must be adjusted to match the CPU set that is specified in the cluster performance profile and must be accurate for the number of cores in the cluster host.
++
+include::snippets/ztp-02-master-workload-partitioning.adoc[leveloffset=+1]
+
+* When configured in the cluster host, the contents of `/etc/crio/crio.conf.d/01-workload-partitioning` should look like this:
++
+[source,terminal]
+----
+[crio.runtime.workloads.management]
+activation_annotation = "target.workload.openshift.io/management"
+annotation_prefix = "resources.workload.openshift.io"
+[crio.runtime.workloads.management.resources]
+cpushares = 0
+CPUs = "0-1, 52-53" <1>
```

Comment:
data source in "Recommended workload partitioning configuration" translates to the following. 
```
[crio.runtime.workloads.management]
activation_annotation = "target.workload.openshift.io/management"
annotation_prefix = "resources.workload.openshift.io"
resources = { "cpushares" = 0, "cpuset" = "0-1,52-53" }
```

Reply:
Updated with `example-sno_machineconfig_02-master-workload-partitioning.yaml` extracted from `ztp-site-generate`